src/Components/bookHub/BookCard.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import React, { useState, useEffect } from 'react';
+
+import './BookCard.css';
+
+// airtable configuration
+const Airtable = require('airtable');
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey })
+  .base(airtableConfig.baseKey);
+
+export default function Card({
+  id, title, author, image,
+}) {
+  const [authorVar, setAuthor] = useState();
+
+  const authorRecords = () => {
+    base('Creator').find(author, (err, record) => {
+      if (err) { console.error(err); }
+      setAuthor(record);
+    });
+  };
+
+  useEffect(authorRecords, []);
+
+  return ( // horizontal scroll not implemented
+    <div className="card">
```

Comment:
One Material UI component that could be very helpful for you here is the [card component](https://mui.com/components/cards/)! It's a component with all of the styling you'd want for these containers of the book information. If you have time, this is a pretty quick change that will make the UI easier!